From ad478d123c422f34da23e2c98f8a92c984dd1db7 Mon Sep 17 00:00:00 2001
Message-ID: <ad478d123c422f34da23e2c98f8a92c984dd1db7.1703031694.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1703031693.git.sharlatanus@gmail.com>
References: <cover.1703031693.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Mon, 18 Dec 2023 23:31:37 +0000
Subject: [PATCH 28/32] gnu: python-synphot: Adjust package style.

* gnu/packages/astronomy.scm (python-synphot): Adjust package style.
[propagated-inputs]: Add annotation for optional, not packed yet
packages. Columnise inputs.
[native-inputs]: Columnise inputs.

Change-Id: I789fc143191c68d5acc248bf6c67e52a93e2744f
---
 gnu/packages/astronomy.scm | 25 ++++++++++++++++---------
 1 file changed, 16 insertions(+), 9 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 2fd7454977..75912cefb1 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -3743,12 +3743,12 @@ (define-public python-synphot
   (package
     (name "python-synphot")
     (version "1.3.0")
-    (source (origin
-              (method url-fetch)
-              (uri (pypi-uri "synphot" version))
-              (sha256
-               (base32
-                "0a54bfrx9aar66040324sw9qdjz5lg67y28sjsxhyv9h3gppc7c0"))))
+    (source
+     (origin
+       (method url-fetch)
+       (uri (pypi-uri "synphot" version))
+       (sha256
+        (base32 "0a54bfrx9aar66040324sw9qdjz5lg67y28sjsxhyv9h3gppc7c0"))))
     (build-system pyproject-build-system)
     (arguments
      (list
@@ -3767,9 +3767,16 @@ (define-public python-synphot
                 (lambda (port)
                   (format port "[pytest]
 python_files = test_*.py"))))))))
-    (propagated-inputs (list python-astropy python-numpy python-scipy))
-    (native-inputs (list python-pytest python-pytest-astropy
-                         python-setuptools-scm))
+    (propagated-inputs
+     (list ;; python-dust-extinction ; XXX: Not packed yet, optional.
+           ;; python-specutils       ; XXX: Not packed yet, optional.
+           python-astropy
+           python-numpy
+           python-scipy))
+    (native-inputs
+     (list python-pytest
+           python-pytest-astropy
+           python-setuptools-scm))
     (home-page "https://github.com/spacetelescope/synphot_refactor")
     (synopsis "Synthetic photometry using Astropy")
     (description
-- 
2.41.0

